docker/Dockerfile

```diff
@@ -28,47 +28,7 @@ ARG GID=1000
 
 # PDH and WEL extensions and not listed as they are Windows specific
 # SYSTEMD extension is turned OFF explicitly as it has no use in an alpine container
```

Comment:
These comments can be removed

Reply:
nice catch, removed this in https://github.com/apache/nifi-minifi-cpp/pull/1369/commits/69bbe6d590f5608d809a28fff7aa062bfd80800b